Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-with-servicenow-playbook.yml

```diff
@@ -0,0 +1,357 @@
+id: PANW IoT Incident Handling with ServiceNow
+version: -1
+name: PANW IoT Incident Handling with ServiceNow
+description: This playbook creates a ServiceNow ticket after the incident enrichment
+  from Palo Alto Networks IoT security portal (previously Zingbox Cloud).
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: b49efade-d3b4-4113-8f8f-8bfa278956f6
+    type: start
+    task:
+      id: b49efade-d3b4-4113-8f8f-8bfa278956f6
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: 82d2db59-0b8e-4e89-8c03-adbecf1eddd9
+    type: title
+    task:
+      id: 82d2db59-0b8e-4e89-8c03-adbecf1eddd9
+      version: -1
+      name: Incident Enrichment
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "2":
```

Comment:
Better to add a conditional task to check if the integration is enabled before.

Reply:
You meant, checking this IoT security XSOAR integration to see if it's enabled?

Reply:
Exactly, otherwise If the integration isn't enabled the task will fail rather than going to done

Reply:
Cool. I add it in the next check in.